app/src/main/res/drawable/ic_sync_disconnected.xml

```diff
@@ -0,0 +1,12 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="23dp"
+    android:height="21dp"
+    android:viewportWidth="23"
+    android:viewportHeight="21">
+  <path
+      android:pathData="M15.9992,9C16.6039,9 17.1604,9.3162 17.4716,9.8263L17.5442,9.9578L21.8642,18.5078C22.1369,19.0569 22.0977,19.7095 21.7612,20.222C21.4527,20.6917 20.937,20.9791 20.3815,20.999L20.2292,20.9978L11.6792,20.9978C11.0909,20.9822 10.5511,20.6677 10.2474,20.1635C9.9741,19.7098 9.9259,19.1588 10.1094,18.6687L10.1792,18.5078L14.4542,9.9578C14.7456,9.3711 15.3441,9 15.9992,9ZM10,0C15.5228,0 20,4.4772 20,10C19.9992,10.7347 19.9192,11.451 19.7682,12.1406L18.4368,9.5068C17.9796,8.5862 17.0341,8 15.9992,8C15.1482,8 14.3575,8.3964 13.8475,9.0527C14.1063,8.5065 14.25,7.895 14.25,7.25C14.25,4.9026 12.3474,3 10,3C7.6526,3 5.75,4.9026 5.75,7.25C5.75,9.5974 7.6526,11.5 10,11.5C11.4196,11.5 12.6765,10.8042 13.4483,9.735L12.2065,12.2178C11.4955,12.5135 10.7527,12.8 9.9997,12.8C7.8675,12.8 5.8151,10.4998 4.3339,12.2092C3.9139,12.6936 3.8803,13.3992 4.2401,13.9284C5.4639,15.7293 7.4894,16.9372 9.8164,16.9976L9.2848,18.0606C8.989,18.6578 8.9251,19.3312 9.0873,19.9574C3.992,19.4977 0,15.215 0,10C0,4.4772 4.4772,0 10,0ZM15.323,17.2915C15.0312,17.5833 14.944,18.022 15.1019,18.4031C15.2597,18.7843 15.6317,19.0328 16.0442,19.0328C16.6075,19.0328 17.0642,18.5761 17.0642,18.0128C17.0642,17.6002 16.8157,17.2283 16.4345,17.0704C16.0534,16.9126 15.6147,16.9998 15.323,17.2915ZM15.5841,11.9901C15.2989,12.1719 15.1449,12.5025 15.1892,12.8378L15.1892,15.4328C15.2464,15.8655 15.6153,16.1888 16.0517,16.1888C16.4881,16.1888 16.857,15.8655 16.9142,15.4328L16.9142,12.8378C16.9585,12.5025 16.8045,12.1719 16.5193,11.9901C16.2341,11.8084 15.8694,11.8084 15.5841,11.9901Z"
+      android:strokeWidth="1"
+      android:fillColor="#20123A"
+      android:fillType="evenOdd"
```

Comment:
We generally stay away from `evenOdd` because it's not supported well by older devices. 
Try replacing this with `nonZero` and see if the icon looks normal! If it doesn't talk to me and I can get us a new icon.